src/SourceBuild/content/build.sh

```diff
@@ -214,13 +214,12 @@ fi
 . "$scriptroot/eng/common/tools.sh"
 
 project="$scriptroot/build.proj"
-targets="/t:Build"
+actions=( "/p:Restore=true" "/p:Build=true" "/p:Publish=true")
 
 # This repo uses the VSTest integration instead of the Arcade Test target
 if [[ "$test" == true ]]; then
   project="$scriptroot/test/tests.proj"
-  targets="$targets;VSTest"
-  properties+=( "/p:Test=true" )
+  actions=( "/p:Restore=true" "/p:Build=true" "/p:Test=true" "/p:IsTestRun=true")
```

Comment:
Isn't this changing the test infra to Test instead of VSTest?

Reply:
You are correct, it is doing that. I'll make sure to update the comment above

Reply:
We have several settings that are VSTest specific:
- https://github.com/dotnet/dotnet/blob/30af19818c3c5be92121daf98f4e109f5109d1e5/test/Directory.Build.props#L6
- https://github.com/dotnet/dotnet/blob/30af19818c3c5be92121daf98f4e109f5109d1e5/test/Microsoft.DotNet.SourceBuild.Tests/Microsoft.DotNet.SourceBuild.Tests.csproj#L9-L12
- https://github.com/dotnet/dotnet/blob/30af19818c3c5be92121daf98f4e109f5109d1e5/test/Microsoft.DotNet.Tests/Microsoft.DotNet.Tests.csproj#L6-L7
- https://github.com/dotnet/dotnet/blob/30af19818c3c5be92121daf98f4e109f5109d1e5/test/Microsoft.DotNet.Installer.Tests/Microsoft.DotNet.Installer.Tests.csproj#L5-L6